ui/src/BulkUploads.js

```diff
@@ -0,0 +1,302 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { convertStatusText } from "./common";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import {
+  Button,
+  CircularProgress,
+  Dialog,
+  DialogContent,
+  LinearProgress,
+  Paper,
+  Snackbar,
+  SnackbarContent,
+  Typography,
+} from "@material-ui/core";
+import TableContainer from "@material-ui/core/TableContainer";
+import Table from "@material-ui/core/Table";
+import TableHead from "@material-ui/core/TableHead";
+import TableBody from "@material-ui/core/TableBody";
+import JobDetails from "./JobDetails";
+import { Link } from "react-router-dom";
+
+class BulkUploads extends Component {
+  state = {
+    jobs: [],
+    authorisedActivities: [],
+    fileProgress: 0, // Percentage of the file uploaded
+    fileUploadSuccess: false, // Flag to flash the snackbar message on the screen, when file uploads successfully
+    uploadInProgress: false, // Flag to display the file upload progress modal dialog
+    showDetails: false, // Flag to display the job details dialog
+  };
+
+  componentDidMount() {
+    this.getAuthorisedBackendData();
+    this.getJobs();
```

Comment:
This is not authorised

Reply:
`getAuthorisedBackendData` is the place to get data which is authorised from the backend, as the name suggests